integration/common.go

```diff
@@ -353,9 +353,16 @@ func getJSONElement(jsonString string, jsonPath string) (string, error) {
 func get(url string) (string, error) {
 	logrus.Println(fmt.Sprintf("\nPerforming GET request to:  %s\n", url))
 
-	resp, err := http.Get(url) // #nosec: testing only.
+	req, err := http.NewRequest(http.MethodGet, url, nil)
 	if err != nil {
-		return "", errors.Wrapf(err, "getting url: %s", url)
+		return "", errors.Wrap(err, "building http req")
+	}
+
+	req.Header.Set("Origin", "http://foo.com")
```

Comment:
did you mean to do this for every get() request?

Reply:
Actually removed this. An actual test would need to check the presence of a header.